wf/__init__.py

```diff
@@ -459,18 +462,18 @@ def deseq2_wf(
             ),
         ]
     ] = None,
-    raw_count_tables: List[FlyteFile] = [],
+    raw_count_tables: List[LatchFile] = [],
     count_table_gene_id_column: str = "gene_id",
     output_location_type: str = "default",
-    output_location: Optional[FlyteDirectory] = None,
+    output_location: Optional[LatchDir] = None,
     conditions_source: str = "manual",
     manual_conditions: Annotated[
```

Comment:
at some point we should probably nix the FlyteAnnotation stuff - I can rework metadata to take in arbitrary arguments and stick them into the docstring appropriately so that stuff like `_tmp_hack_deseq` would still work

Reply:
will open an issue for this